packages/core/build/tokens.ts

```diff
@@ -12,6 +12,7 @@ const layout = {
     xs: token(4),
     sm: token(6),
     md: token(12),
+    xmd: token(18),
```

Comment:
In order to follow the design spces for the spaces in the card we need to introduce a new t-shirt size. 

We decided to go in this direction cause the other way is to provide CSS styles that you will need to add when your application is using the card component and this will complicate the things a lot.

Reply:
@colinreedmiller 
Why are we already extending our spacing tokens? We did a lot of research on this and I believed we were confident in our design token sizes.

This feels like a slippery slope and we are having to make special cases here just to avoid a pretty major breaking change. 

It's a little disappointing to run across this less than a year into the design tokens.

Reply:
The research into T-shirt sizes did not include design and the result omitted some necessary sizes. I put in the request to make some adjustments as necessary and was informed that though we had performance concerns, and that these adjustments were possible, especially in cases where we were lacking in standard sizes. There seems to be a communication disconnect between where spacing tokens are and where they need to be relative to what design is requesting.

Reply:
Something to note on this there is a 18px spacing token already in the system `--cds-global-space-8` The system has two sets of tokens. [Space Tokens](https://clarity.design/storybook/core/?path=/story/foundation-spacing--page#component-spacing) and [Layout Tokens](https://clarity.design/storybook/core/?path=/story/foundation-spacing--page#application-layout-spacing). Space provides more values for more specific styles in a component. The layout values are specific to the whitespace between components. This distinction is important because for compact/large touch target themes to work properly they need to be tread as separate scales. Performance wise the layout values are limited due to the amount of utility CSS they generate.

Look at [this example](https://clarity.design/storybook/core/?path=/story/themes-dynamic-themes--page) of why the two scales work together. Adjust the scale and density options.

A question on the API design why does the Card not retain its own padding but require wrapper elements around the content?

Reply:
@coryrylan We thought about the scenario you are talking about and I documented our thoughts [here](https://app.clubhouse.io/clarity-design-system/story/114/card-component)

The `slotted` selector can only select direct children of the element. In our case the direct child of `cds-card` is the `div.private-host`.